File: solutions/2624-difference-between-element-sum-and-digit-sum-of-an-array/solution.py
class Solution(object):
    def differenceOfSum(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        ls=sum(nums)
        s=""
        for i in nums:
            s+=str(i)
        n=0
        for i in s:
            n+=int(i)
        return abs(ls-n)

File: solutions/2624-difference-between-element-sum-and-digit-sum-of-an-array/solution.c
int differenceOfSum(int* nums, int numsSize) {
    int sum1=0,sum2=0;
    for(int i=0;i<numsSize;i++)
    {
        sum1+=*(nums+i);
        while(*(nums+i)!=0)
        {
            sum2+=(*(nums+i)%10);
            *(nums+i)/=10;
        }
    }
    return abs(sum2-sum1);
}
